src/main/java/Task.java

```diff
@@ -0,0 +1,19 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
isDone could be in ToDo instead of Task because only ToDo uses it and not the other stuff like Deadline and Event